data-prepper-api/src/main/java/org/opensearch/dataprepper/logging/DataPrepperMarkers.java

```diff
@@ -6,9 +6,11 @@
 public final class DataPrepperMarkers {
     public static final Marker EVENT = MarkerFactory.getMarker("EVENT");
     public static final Marker SENSITIVE = MarkerFactory.getMarker("SENSITIVE");
+    public static final Marker NOISY = MarkerFactory.getMarker("NOISY");
 
     static {
         EVENT.add(SENSITIVE);
+        NOISY.add(SENSITIVE);
```

Comment:
I don't think we want this relationship. A log may be noisy without being sensitive.